### ISSUE TITLE
my.gobluefire.com

### ISSUE THREAD

**Issue by AdamWr** [AdguardTeam/CoreLibs#1288]
@adguard-bot commented on [Thu Apr 09 2020](https://github.com/AdguardTeam/AdguardFilters/issues/53280)

### Issue URL (Incorrect Blocking)
 [https://my.gobluefire.com/app/giving/havenfellowship](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmy.gobluefire.com%2Fapp%2Fgiving%2Fhavenfellowship)
### Comment
> Website doesn't show up if Adguard is enabled. I'm whitelisting it for now on my computer, but I'm afraid there will be others who won't be able to view the site to donate if it isn't fixed. Thanks
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> 
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.meta.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js
> 
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  ![Screenshot 1](https://reports-img.adguard.com/uRYyIqj.png)
</details>
<details>
  <summary>Screenshot 2</summary>

  ![Screenshot 2](https://reports-img.adguard.com/YsvXL2E.png)
</details>

### System configuration

Information | value
--- | ---
Platform: | Mac
AdGuard version: | 2.3.2.710 release
Browser: | Chrome
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (180), Block Push API, Block Location API, Block Java
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>Icelandic ABP List
Custom filter subscriptions: |

**Comment by AdamWr** [comment]
It seems that on this website should be an iframe from `forms.ministryforms.net`, but it looks like it's broken by "Extension" module.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/78980078-6b23d580-7b1d-11ea-8cda-f6d2e14fb0aa.png)

</details>

Before checking it, please disable this rule from `AdGuard Base` filter:
`@@||forms.ministryforms.net/embed.aspx?formId=$extension`

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/CoreLibs/issues/1222
